packages/redux/src/bags/types/state.types.ts

```diff
@@ -12,11 +16,18 @@ export type BagItemsState = CombinedState<{
   [k: string]: unknown;
 }>;
 
+export type BagOperationsState = CombinedState<{
+  error: Record<BagOperation['id'], BlackoutError | null>;
+  isLoading: Record<BagOperation['id'], boolean>;
+  [k: string]: unknown;
```

Comment:
This can be removed since the `BagOperationsState` will only contain the `error` and `isLoading` properties according to its reducer.

Reply:
Done!